question-generation/comments-wrangling/src/taskpane/taskpane.js

```diff
@@ -29,3 +43,27 @@ export async function tryCatch(callback) {
     console.error(error);
   }
 }
+
+async function main() {
+  await Word.run(async (context) => {
+    const commentCollection = context.document.body.getComments();
+    commentCollection.load("items");
+    await context.sync();
+
+    const contexts = await getCommentContexts(commentCollection);
+    const replies = await getCommentReplies(commentCollection);
+
+    const data = [];
+
+    for (let i = 0; i < commentCollection.items.length; i++) {
+      const comments = [commentCollection.items[i].content];
+      if (replies[i].length > 0) {
+        comments.push(...replies[i]);
```

Comment:
isn't this `comments.concat(replies)`? I don't have Javascript in my fingers though